cmd/operator-sdk/alpha/kubebuilder/cmd.go

```diff
@@ -15,62 +15,97 @@
 package kubebuilder
 
 import (
-	"fmt"
+	"os"
 	"os/exec"
 
-	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/generate"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
-
-	"github.com/google/shlex"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-var kbFlags string
-
 //nolint:lll
-func NewCmd() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "kubebuilder [init/create cmds]",
-		Short: "Kubebuilder aligned subcommands",
-		Long: `
+const longHelp = `
 This subcommand is a placeholder to test the integration of Kubebuilder and Operator SDK
 subcommands until a plugin system is available to integrate the Kubebuilder CLI.
-See: https://github.com/kubernetes-sigs/kubebuilder/pull/1250
-and https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/openshift-4.4/kubebuilder-integration.md
-`,
-		RunE:   runKubebuilder,
-		Hidden: true,
-
-		Example: `
-	# Initialize your project
-	operator-sdk alpha kubebuilder init --kb-flags="--domain example.com --license apache2 --owner \"The Kubernetes authors\""
-
-	# Create a frigates API with Group: ship, Version: v1beta1 and Kind: Frigate
-	operator-sdk alpha kubebuilder create api --kb-flags="--group ship --version v1beta1 --kind Frigate"
-`,
+
+See:
+Proposal: https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/kubebuilder-integration.md
+Design doc: https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-1.md
+
+Kubebuilder version v2.0.0 or above must be installed to run this command:
+https://book.kubebuilder.io/quick-start.html
+`
+
+func runKubebuilder(args ...string) {
+	kbCmd := exec.Command("kubebuilder", args...)
+	if err := projutil.ExecCmd(kbCmd); err != nil {
+		os.Exit(1)
+	}
+}
+
+func newKubebuilderRunFunc(cmds ...string) func(*cobra.Command, []string) {
+	return func(_ *cobra.Command, args []string) {
+		if _, err := exec.LookPath("kubebuilder"); err != nil {
+			log.Fatal(err)
+		}
+		runKubebuilder(append(cmds, args...)...)
+	}
+}
+
+// Commands are all kubebuilder commands the alpha command will support.
+var Commands = []*cobra.Command{
+	newInitCmd(),
+	newCreateCmd(),
+}
+
+func newInitCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:                "init",
+		Short:              `kubebuilder's "init" subcommand`,
+		Long:               longHelp,
+		Run:                newKubebuilderRunFunc("init"),
+		Hidden:             true,
```

Comment:
How do we feel about these continuing to be hidden? I'm thinking we want to expose these commands so people begin to migrate towards a kubebuilder-style CLI.

Reply:
Let's keep these hidden for now.
If we advertise init and create cmds as being ready, users still can't migrate towards a kubebuilder-style CLI until we have the rest of the commands aligned and the docs updated.
We can switch over and unhide the cmds once everything is ready and lined up for a release.